procedures/foreman/branch.md.erb

```diff
@@ -89,7 +89,12 @@
 * Create Redmine versions
   * [ ] Add next version number (<%= develop %>) and make sure it is shared with subprojects in [foreman](https://projects.theforeman.org/projects/foreman/settings/versions)
   * [ ] Add first patch release (<%= release %>.1) and make sure it is shared with subprojects in [foreman](https://projects.theforeman.org/projects/foreman/settings/versions)
-* [ ] Create test_<%= release.tr('.', '_') %>_stable.yaml and test_proxy_<%= release.tr('.', '_') %>_stable.yaml for [JJB in foreman-infra](https://github.com/theforeman/foreman-infra/tree/master/puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs), remove the oldest version to keep last 3
+* [ ] Add the new release to the JJB job definitions in [foreman-infra](https://github.com/theforeman/foreman-infra/tree/master/puppet/modules/jenkins_job_builder/files/theforeman.org/)
+  * [ ] Create a `<%= release %>.groovy` in `pipelines/vars/foreman/` describing the operating systems the release supports
+  * [ ] Add `<%= release %>` to the version list in `yaml/jobs/pipeline/foreman-release.yaml`
+  * [ ] Add `<%= release %>` to the version list in `pipelines/release/foremanPluginsRelease.groovy`
```

Comment:
Going to hijack this with a todo list. @ehelms and I have talked about creating a plugin pipeline per release. That could be driven from ``yaml/jobs/pipeline/foreman-release.yaml` so you automatically get a correct release pipeline. This will be especially useful when we add EL8 support since then it'll depend on the vars file.

We should also change the `test_$release` and `test_proxy_$release` jobs to pipelines fitting the naming scheme. So `foreman-1.24-stable-source` and `smart-proxy-1.24-stable-source`. Again, these should be driven by `yaml/jobs/pipeline/foreman-release.yaml`.

Obviously not blocking the merge of this.